example/src/App.tsx

```diff
@@ -1,51 +1,84 @@
-import { NavigationContainer } from '@react-navigation/native';
```

Comment:
This is a good example for barcode scanning, but can you revert that to the original example app without Skia / Canvas code? Thank you!